psi4/src/export_mints.cc

```diff
@@ -348,30 +347,35 @@ void export_mints(py::module& m) {
         .def("begin", &Slice::begin, "Get the first element of this slice")
         .def("end", &Slice::end, "Get the past-the-end element of this slice");
 
-    py::class_<Vector, std::shared_ptr<Vector>>(m, "Vector", "Class for creating and manipulating vectors",
+    py::class_<IrrepedVector<double>, std::shared_ptr<IrrepedVector<double>>>(m, "ProtoVector");
+
+    py::class_<Vector, std::shared_ptr<Vector>, IrrepedVector<double>>(m, "Vector", "Class for creating and manipulating vectors",
                                                 py::dynamic_attr())
         .def(py::init<int>())
         .def(py::init<const Dimension&>())
         .def(py::init<const std::string&, int>())
         .def(py::init<const std::string&, const Dimension&>())
-        .def_property("name", py::cpp_function(&Vector::name), py::cpp_function(&Vector::set_name),
+        .def_property("name", &Vector::name, &Vector::set_name,
                       "The name of the Vector. Used in printing.")
+        .def("init", &Vector::init, "Reallocate the data of the Vector. Consider making a new object.")
         .def("get", vector_getitem_1(&Vector::get), "Returns a single element value located at m", "m"_a)
         .def("get", vector_getitem_2(&Vector::get), "Returns a single element value located at m in irrep h", "h"_a,
              "m"_a)
         .def("set", vector_setitem_1(&Vector::set), "Sets a single element value located at m", "m"_a, "val"_a)
         .def("set", vector_setitem_2(&Vector::set), "Sets a single element value located at m in irrep h", "h"_a, "m"_a,
              "val"_a)
-        .def("copy", vector_one(&Vector::copy), "Returns a copy of the matrix")
+        .def("add", vector_setitem_1(&Vector::add), "Add to a single element value located at m", "m"_a, "val"_a)
+        .def("add", vector_setitem_2(&Vector::add), "Add to a single element value located at m in irrep h", "h"_a, "m"_a,
+             "val"_a)
+        .def("copy", &Vector::copy, "Copy another vector into this.")
         .def(
             "clone",
             [](Vector& vec) {
-                std::shared_ptr<Vector> result = std::move(vec.clone());
-                return result;
+                return std::make_shared<Vector>(std::move(vec.clone()));
             },
             "Clone the vector")
         .def("zero", &Vector::zero, "Zeros the vector")
-        .def("print_out", &Vector::print_out, "Prints the vector to the output file")
```

Comment:
`vector.print_out()` is literally the same thing as `vector.print()` in the old code. I don't understand why we need `print_out`, since it doesn't _also_ offer the optional arguments that `print` does. Do we want `print_out` for consistency across different `libmints` objects?